### PR TITLE
added clock component, extra style

### PR DESCRIPTION
**Pull request by GedasIncirauskas**
1. Created clock component.
2. Added/fixed style at app.

Screenshot 2022-06-13 120311

### CODE REVIEW

File: src/components/Clock/Clock.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useState, useEffect } from 'react';
+import * as S from './Clock.styles';
+
+const Clock = () => {
+  const [date, setDate] = useState(new Date());
+  const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```
Comment (JuliaShavyrova) [inline src/components/Clock/Clock.jsx]: please get weekday names using js methods

File: src/components/Clock/Clock.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  };
+
+  useEffect(() => {
+    const timerId = setInterval(refreshClock, 1000);
+    return () => clearInterval(timerId);
+  }, []);
+
+  return (
+    <S.Wrapper>
+      <h1>{date.toLocaleTimeString('lt-LT')}</h1>
```
Comment (JuliaShavyrova) [inline src/components/Clock/Clock.jsx]: just a suggestion: you could get info about user language by using browser global vars

Comment (GedasIncirauskas) [reply src/components/Clock/Clock.jsx]: Thank you, fixed 👍

File: src/components/Clock/Clock.jsx (hunk 1 above)
Comment (GedasIncirauskas) [reply src/components/Clock/Clock.jsx]: Ok, fixed.

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**